install/jakartaee/other/vehicle.properties

```diff
@@ -88,7 +88,7 @@ com/sun/ts/tests/javamail = appclient ejb servlet jsp
 com/sun/ts/tests/samples/javamail = ejb servlet jsp
 com/sun/ts/tests/samples/jdbc = ejb web appclient
 
-com/sun/ts/tests/signaturetest/javaee = ejb servlet jsp appclient standalone
```

Comment:
For the record, the standalone vehicle is removed from https://github.com/eclipse-ee4j/jakartaee-tck/commit/36bbfc810a49e9c4adb6be432b4aaeea00e0e95c as the tests were only for deployment jars.